### PR TITLE
Changed LoadImages method that returns raw images bytes to LoadRawImageBytes

### PR DESCRIPTION
**Pull request by harshithapv**
Changed LoadImages method that returns raw images bytes to LoadRawImageBytes.
This addresses issue #4313

### CODE REVIEW

**Comment by eerhardt** [comment]
>         /// Create a <see cref="ImageLoadingEstimator"/>, which loads the data from the column specified in <paramref name="inputColumnName"/>

The summary should say what is different about this method than the `LoadImages` above. Right now they say the same thing.

---
Refers to: src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs:75 in b827c39. [](commit_id = b827c390f6b44ba7e05b421c88a11281182a8b82, deletion_comment = False)

**Comment by eerhardt** [comment]
ImageLoadingTransformer has this property:

https://github.com/dotnet/machinelearning/blob/9bfc3c12965c664ca75db1420cde7fea06c2342a/src/Microsoft.ML.ImageAnalytics/ImageLoader.cs#L76-L80

it should be internal.

Any references to `useImageType` should be made internal.

File: test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs [hunk 1]
```diff
@@ -1243,11 +1243,11 @@ public void TensorFlowImageClassificationDefault()
 
             IDataView trainDataset = trainTestData.TrainSet;
             IDataView testDataset = trainTestData.TestSet;
-            var validationSet = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, false, "ImagePath") // false indicates we want the image as a VBuffer<byte>
+            var validationSet = mlContext.Transforms.LoadRawImageBytes("Image", fullImagesetFolderPath, "ImagePath") // false indicates we want the image as a VBuffer<byte>
```
Comment (eerhardt) [inline test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs]: out of date comments at the end

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs [hunk 2]
```diff
@@ -59,7 +59,7 @@ public static void Example()
                 IDataView trainDataset = trainTestData.TrainSet;
                 IDataView testDataset = trainTestData.TestSet;
                 
-                var validationSet = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, false, "ImagePath") // false indicates we want the image as a VBuffer<byte>
+                var validationSet = mlContext.Transforms.LoadRawImageBytes("Image", fullImagesetFolderPath, "ImagePath") // false indicates we want the image as a VBuffer<byte>
```
Comment (eerhardt) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs]: out of date comments at the end

**Comment by codemzs** [comment]
agreed.

---
In reply to: [546531972](https://github.com/dotnet/machinelearning/pull/4390#issuecomment-546531972) [](ancestors = 546531972)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs (hunk 2 above)
Comment (harshithapv) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningEarlyStopping.cs]: My recent push will address this.

File: src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs [hunk 3]
```diff
@@ -73,23 +73,22 @@ public static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, s
 
         /// <summary>
         /// Create a <see cref="ImageLoadingEstimator"/>, which loads the data from the column specified in <paramref name="inputColumnName"/>
-        /// as an image to a new column: <paramref name="outputColumnName"/>.
+        /// as an image of raw bytes type to a new column: <paramref name="outputColumnName"/>.
```
Comment (codemzs) [inline src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs]: >raw bytes type [](start = 27, length = 14)

just raw bytes

**Pull request merged**